src/utils.py

```diff
@@ -48,7 +48,11 @@
 if TYPE_CHECKING:
     from predicators.src.envs import BaseEnv
 
+# Switch to a non-GUI backend and disable all built-in keyboard shortcuts.
 matplotlib.use("Agg")
+keymaps = {keymap for keymap in plt.rcParams if keymap.startswith("keymap.")}
+for keymap in keymaps:
+    plt.rcParams[keymap].clear()
```

Comment:
to stop keyboard buttons from doing things like opening the save menu or quitting